packages/web/src/common/store/cache/collections/commonSagas.ts

```diff
@@ -213,8 +213,23 @@ function* confirmEditPlaylist(
         )
 
         if (playlistBeforeEdit?.is_private && !confirmedPlaylist.is_private) {
-          for (const track of confirmedPlaylist.tracks ?? []) {
-            if (track.is_unlisted) {
+          const playlistTracks = yield* select(getCollectionTracks, {
+            id: playlistId
+          })
+
+          // Publish all hidden tracks
+          // If the playlist is a scheduled release
+          //    AND all tracks are scheduled releases, publish them all
```

Comment:
Indentation?

Reply:
I did it on purpose to make it clear that it was part of the condition, but I can see either way